app/views/users/edit.html.erb

```diff
@@ -0,0 +1,39 @@
+
+
+<div class="box5">
+<div class="row">
+	<div class="offset-md-3 col-md-6">
+		<h1 class="text-center">Edit</h1>
+
+		<%= form_for @user do |f| %>
+
```

Comment:
@YuheiTsunoda Add error messages here, create partial file for `shared/_error_messages.html.erb`, and move errors from `users/new.html.erb`